Framework/Crystal/inc/MantidCrystal/CentroidPeaks.h

```diff
@@ -38,6 +38,8 @@ class DLLExport CentroidPeaks : public API::Algorithm {
   void integrate();
   void integrateEvent();
   int findPixelID(std::string bankName, int col, int row);
+  void removeEdgePeaks(Mantid::DataObjects::PeaksWorkspace_sptr &peakWS,
```

Comment:
Instead of passing the address of the shared pointer could we just do:
`void removeEdgePeaks(DataObjects::PeaksWorkspace &peaksWS, ...)`
Then the caller just does
`removeEdgePeaks(ws.get());`
Which removes another layer of indirection too